src/eckit/geo/range/RegularLatitude.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * (C) Copyright 1996- ECMWF.
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * In applying this licence, ECMWF does not waive the privileges and immunities
+ * granted to it by virtue of its status as an intergovernmental organisation nor
+ * does it submit to any jurisdiction.
+ */
+
+
+#pragma once
+
+#include "eckit/geo/range/Regular.h"
+
+
+namespace eckit::geo::range {
+
+
+class RegularLatitude final : public Regular {
+public:
+    // -- Constructors
+
+    explicit RegularLatitude(double inc, double a, double b, double ref, double eps = 0.);
```

Comment:
What is the use of the ref argument?

Reply:
When you define a regional regular_ll grid with area=n/w/s/e, the points will include the point defined at w/s by default. The ref defines a different point to include (or +- increments until it is included in area)

This is to support "shifted" and "non-shifted" grids for the same definition of area and increments.

Reply:
Could this be documented in code for future readings.